src/templates/partials/common-includes.handlebars

```diff
@@ -0,0 +1,167 @@
+<!-- <script type="text/javascript" src="/modules/gpii-express-user/src/js/client/signup.js"></script> -->
+<!-- <script type="text/javascript" src="/modules/gpii-express-user/src/js/client/resend.js"></script> -->
+<!-- <link rel="stylesheet" href="/modules/gpii-express-user/src/css/user-controls.css"> -->
+<!-- <link rel="stylesheet" href="/modules/gpii-express-user/src/css/user-icons.css"> -->
+
+
+<!-- include foundation and its dependencies -->
+<script src="/modules/foundation-sites/dist/js/foundation.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.core.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.util.keyboard.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.util.box.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.util.mediaQuery.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.util.motion.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.util.triggers.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.responsiveMenu.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.responsiveToggle.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.sticky.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.tooltip.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.reveal.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.accordion.js"></script>
+
+<!-- TODO: Verify if these are necessary -->
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.dropdown.js"></script>
+<script src="/modules/foundation-sites/dist/js/plugins/foundation.dropdownMenu.js"></script>
+
+<!-- Bring in the query string encoding we use in our client-side components -->
+<script src="/modules/gpii-express/src/js/lib/querystring-coding.js"></script>
+
+<!-- <link rel="stylesheet" href="/modules/foundation-sites/dist/css/foundation.css"> -->
+<link rel="stylesheet" href="/src/css/app.css" />
+<link rel="stylesheet" href="/src/css/foundation-icons/foundation-icons.css" />
+
+<script src="/modules/lunr/lunr.js"></script>
+<script src="/modules/file-saver/FileSaver.js"></script>
+
+<!-- zxcvbn -->
+<script src="/modules/zxcvbn/dist/zxcvbn.js"></script>
+
+
+<!-- include our common code -->
+<link rel="stylesheet" type="text/css" href="/src/css/devpmt.css" />
+<script src="/src/js/common/util.js"></script>
+<script src="/src/js/client/common-event-bindings.js"></script>
+<script src="/src/js/client/common-jsondev-widget.js"></script>
+
+<!-- Include a raw set of message bundles until we have the i18n tooling in gpii-handlebars -->
+<!--<script src="/src/messageBundles/devpmt-messageBundle_en.js"></script>-->
```

Comment:
This inclusion should be on a json file ending with ".json" rather than ".js".

Reply:
This is json now with the proper gpii-handlebars usage